### ISSUE TITLE
Dynamic playback screen coloring

### ISSUE THREAD

**Issue by OxygenCobalt** [OxygenCobalt/Auxio#764]
I've rejected this prior, but 3 developments have occurred that have justified this:

1. It's now an official recommendation of the Material3 spec
2. MDC now has a way to extract color palettes from images using Monet (I think...)
3. #743 is making the playback view more likely to play along with this.
4. I think I have a way to implement this technically that would not be an absolute nightmare (Basically I have a mini shared object scoped to the fragment lifecycle that hooks into the image loader and animates the colors across all UI components that subscribe to it)

This is likely to be made an unchangeable default, and it will be solely scoped to the playback screen to keep myself sane. Technical/performance implications need to be worked out however.

**Comment by KraXen72** [comment]
HUGE IF TRUE

**Comment by HyperWinX** [comment]
Omg, we will wait for it!

**Comment by OxygenCobalt** [comment]
Update: Apparently MDC's palette extraction uses a weird hacky theming thing that only works on Android 12. I'll have to use a third-party library still to make it work on older android versions and actually properly animate it.

**Comment by KraXen72** [comment]
https://github.com/AkaneTan/Gramophone has implemented this and is open source. that is all i'll say.

**Comment by OxygenCobalt** [comment]
Looking at the source code... yeah this is easier than I thought. Huh.

**Comment by OxygenCobalt** [comment]
Looking at the source code... yeah this is easier than I thought. Huh. @KraXen72 

Still only limited to Android 12 by design, frustratingly.

**Comment by KraXen72** [comment]
there are more things in that music player that could 'inspire' you, if you want

their now-playing screen is pretty clean:
![Screenshot_20240519-011736_Gramophone.png](https://github.com/OxygenCobalt/Auxio/assets/21956756/3ad0486c-cff1-4010-99b8-2b934151c2f9)

**Comment by OxygenCobalt** [comment]
Yeah, it looks better @KraXen72. I actually used it as partial inspiration over at #743, but I disagree with the button placement.

**Comment by KraXen72** [comment]
yeah the bottom button placement is pretty cluttered there.
